deployment/kubernetes/charts/origin/templates/growth.deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "growth.fullname" . }}
```

Comment:
This needs to be defined in `_helpers.tpl`. Same thing for `growth.host`. You can just copy and paste them for another service.

Reply:
ah yes. thank you for catching this.